Shared/Samples/Configure basemap style parameters/ConfigureBasemapStyleParametersView.swift

```diff
@@ -0,0 +1,143 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ConfigureBasemapStyleParametersView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The selected basemap style language strategy.
+    @State private var selectedLanguage: BasemapStyleLanguage = .global
+    
+    /// The selected locale.
+    @State private var selectedLocale: Locale = .current
+    
+    var body: some View {
+        MapView(map: model.map)
+            .toolbar {
+                ToolbarItem(placement: .bottomBar) {
+                    languageMenu
+                        .onChange(of: selectedLanguage) { newValue in
+                            model.setBasemapStyleParameters(language: newValue)
+                        }
+                }
```

Comment:
```suggestion
                ToolbarItemGroup(placement: .bottomBar) {
                    Spacer()
                    languageMenu
                        .onChange(of: selectedLanguage) { newValue in
                            model.setBasemapStyleParameters(language: newValue)
                        }
                    Spacer()
                }
```

I think this makes it look better on Mac Catalyst as it prevents the "Specific Language" picker text from being cut off.